apps/web/src/hooks/latte/helpers.ts

```diff
@@ -1,12 +1,12 @@
-import type { ToolCall } from '@latitude-data/constants/legacyCompiler'
 import { LatteTool } from '@latitude-data/constants/latte'
 import { LatteToolStep } from './types'
+import { ToolRequestContent } from '@latitude-data/compiler'
```

Comment:
Do not import from compiler please, this should be in legacy compiler and if not move it there